androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java

```diff
@@ -177,7 +177,7 @@ public void test1_check_certif_not_secure_no_accept()
 
             // Check that login button keeps on being disabled
```

Comment:
Same here, `// Check that login button keeps on not being displayed`

Reply:
Done!